### DISCUSSION TITLE
Add audio narration support for PowerPoint

### DISCUSSION THREAD

**Discussion by mcraiha** [marp-team/marp#439] [Ideas] [answered]
I have been looking for a tool to create video presentations. Marp doesn't have video output support, but if it would support audio narrations in PowerPoint output mode, I could then use PowerPoint to create the final video file. 

So something like
`![narration](audio.mp3)`

**Comment by yhatt** [comment, marked as answer]
We have no plans for that because [PptxGenJS](https://github.com/gitbrent/PptxGenJS), the library for generating pptx document in Marp ecosystem (CLI and VS Code extension), is not getting support for audio narrations.
